specs/_features/focil/validator.md

```diff
@@ -0,0 +1,116 @@
+# FOCIL -- Honest Validator
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+  - [Time parameters](#time-parameters)
+- [Protocol](#protocol)
+  - [`ExecutionEngine`](#executionengine)
+- [New inclusion list committee assignment](#new-inclusion-list-committee-assignment)
+  - [Lookahead](#lookahead)
+- [New proposer duty](#new-proposer-duty)
+  - [Block proposal](#block-proposal)
+    - [Update execution client with inclusion lists](#update-execution-client-with-inclusion-lists)
+- [New inclusion list committee duty](#new-inclusion-list-committee-duty)
+    - [Constructing a local inclusion list](#constructing-a-local-inclusion-list)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This document represents the changes to be made in the code of an "honest validator" to implement FOCIL.
+
+## Prerequisites
+
+This document is an extension of the [Electra -- Honest Validator](../../electra/validator.md) guide.
+All behaviors and definitions defined in this document, and documents it extends, carry over unless explicitly noted or overridden.
+
+All terminology, constants, functions, and protocol mechanics defined in the updated Beacon Chain doc of [FOCIL](./beacon-chain.md) are requisite for this document and used throughout.
```

Comment:
```suggestion
All terminology, constants, functions, and protocol mechanics defined in the updated Beacon Chain doc of [FOCIL](./beacon-chain.md) are prerequisite for this document and used throughout.
```

Reply:
Sticking with requisite because that's what's used in all the documents, not really sure which is better